influxdb-backup/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM influxdb:1.2.4
+
+# Backup the following databases, seperator ":"
+ENV DATABASES=demo:_internal:testdb
+
+#default InfluxDB host
+ENV INFLUX_HOST=influxdb
+
+# Change workdir
+WORKDIR "/tmp"
```

Comment:
`/tmp` is special, and not a good place for a work directory.  `/opt/influxdb-backup/` is a better place.  In addition, you're not using any unqualified paths below, so this is redundant.